dashboard/app/views/levels/_level_group.haml

```diff
@@ -46,13 +46,27 @@
         = render partial: 'levels/free_response', locals: {index: index, level: level, last_attempt: last_attempt}
 
   - unless @script_level.nil?
+    - if page.page_number > 1
+      %a.btn.btn-large.btn-primary.previousPageButton
+        =t('previous_page')
+      &nbsp;
+      &nbsp;
     - if page.page_number < @total_page_count
       %a.btn.btn-large.btn-primary.nextPageButton
         =t('next_page')
     - else
       = render partial: 'levels/dialog', locals: {app: app, data: data}
-      %a.btn.btn-large.btn-primary.unsubmitButton{style: "display: none"}
-        =t('unsubmit')
+      .level_group_buttons
+        %a.btn.btn-large.btn-primary.unsubmitButton{style: "display: none"}
+          =t('unsubmit')
```

Comment:
That sounds reasonable, but I'd like to do that in a fresh PR if you can wait a bit.  Currently updating & testing this PR and hoping to have it ready for merge soon.